### ISSUE TITLE
Any prebuilt weights available?

### ISSUE THREAD

**Issue by xsacha** [MccreeZhao/QAMFace#1]
I notice you only went to epoch 22, which is still stage 1. Are you still training this?

**Comment by MccreeZhao** [comment]
Hi, thanks for your question.
I wonder what does stage 1 means? The learning rate is decayed in epoch [4, 10, 15,20]. And the model is already converged according to the training curve.

**Comment by xsacha** [comment]
Ah, quick to train!

**Comment by xsacha** [comment]
Is it possible to provide that pytorch weight file?

**Comment by xsacha** [comment]
@MccreeZhao 
Is this what the training graph should look like at epoch 6? From your version.
![image](https://user-images.githubusercontent.com/61218/74588902-50b50a80-504c-11ea-82a0-9d41b3121a09.png)

**Comment by xsacha** [comment]
I was unable to reproduce your results.
My best epochs: 12 / 17
LFW: 99.78 / 99.63
CFP_FF: 99.7 / 99.7
AGEDB: 97.75 / 97.95
